README.md

```diff
@@ -49,58 +51,76 @@ git clone https://github.com/o3de/o3de.git
         *   The `LY_WWISE_INSTALL_PATH` CMake cache variable -- this is checked first
         *   The `WWISEROOT` environment variable which is set when installing Wwise SDK
 
-### Quick Start Build Steps
+For more details and the latest requirements, refer to [System Requirements](https://o3de.org/docs/welcome-guide/requirements/) in the documentation.
+
+### Quick start engine setup
 
-1.  Create a writable folder to cache 3rd Party dependencies. You can also use this to store other redistributable SDKs.
+To set up a project-centric source engine, complete the following steps. For other build options, refer to [Setting up O3DE from GitHub](https://o3de.org/docs/welcome-guide/setup/setup-from-github/) in the documentation.
+
+1.  Create a writable folder to cache downloadable packages. You can also use this to store other redistributable SDKs.
     
-1.  Install the following redistributables to the following:
-    - Visual Studio and VC++ redistributable can be installed to any location
-    - CMake can be installed to any location, as long as it's available in the system path
+1.  Install the following redistributables:
+    - Visual Studio and VC++ redistributable can be installed to any location.
+    - CMake can be installed to any location, as long as it's available in the system path.
 
-1.  Configure the source into a solution using this command line, replacing <your build path> and <3rdParty cache path> to a path you've created:
+1.  Configure the engine source into a solution using this command line, replacing `<your build path>`, `<your source path>`, and `<3rdParty cache path>` with the paths you've created:
+    ```
+    cmake -B <your build path> -S <your source path> -G "Visual Studio 16" -DLY_3RDPARTY_PATH=<3rdParty cache path> -DLY_UNITY_BUILD=ON
     ```
-    cmake -B <your build path> -S <your source path> -G "Visual Studio 16" -DLY_3RDPARTY_PATH=<3rdParty cache path> -DLY_UNITY_BUILD=ON -DLY_PROJECTS=AutomatedTesting 
+    
+    Example:
+    ```
+    cmake -B C:\o3de\build\windows_vs2019 -S C:\o3de -G "Visual Studio 16" -DLY_3RDPARTY_PATH=C:\o3de-packages -DLY_UNITY_BUILD=ON
```

Comment:
FYI Unity build is on by default now.
So the `-DLY_UNITY_BUILD=ON` can be omitted

Reply:
Done.